resolver.go

```diff
@@ -72,7 +72,7 @@ func (r *Resolver) Run(tmpl Templater, w Watcherer) (ResolveEvent, error) {
 
 	switch { // specific to general
 	case err == ErrNoNewValues:
-		return ResolveEvent{}, nil
+		return ResolveEvent{Complete: true, Contents: []byte{}}, nil
```

Comment:
Instead of checking for empty content, I'd like to suggest a new attribute added to `ResolveEvent` that's dedicated to denote there are no new values. I believe 0 length content is a valid render of a template.

Reply:
I had considered 0 length to be a valid render of a template. That's a good point! Thank you 🙏 . I like your idea! Let me think of a name for the new attribute and push a change.

Reply:
I went with `NoChange`. If you have any feedback, let me know!

Reply:
`NoChange` seems fine with me! I like that the default is false, and so there's little impact to existing code usage.

Reply:
If you don't mind, can you update the PR summary to reflect the new approach?

Reply:
Yes! thank you for the reminder. I updated PR summary with the new commit message!